src/test/java/com/iexec/sms/secret/web2/Web2SecretServiceTests.java

```diff
@@ -46,6 +45,7 @@ class Web2SecretServiceTests {
     private EncryptionService encryptionService;
 
     @InjectMocks
+    @Spy
```

Comment:
Why do you need spy?

Reply:
We don't. I have fiddled with the tests to get something working and forgot to remove the `@Spy` annotation. Fixed in https://github.com/iExecBlockchainComputing/iexec-sms/pull/140/commits/50a3804c3a06222a675dca5a4ce811ce2a8a1754.